src/sbp.c

```diff
@@ -348,26 +356,40 @@ void log_(u8 level, const char *msg, ...)
 
 void log_obs_latency(float latency_ms)
 {
-  last_obs_msg_ticks = chVTGetSystemTime();
-
+  float obs_period_ms;
+  systime_t now = chVTGetSystemTime();
+  obs_period_ms = (now - last_obs_msg_ticks) / (double)CH_CFG_ST_FREQUENCY * 1000;
```

Comment:
Did originally, but then realized I had a bug when last_obs_msg_ticks was zero.  now I initialize to zero and only compute if i'm not in the first time through the loop.